config/nstack.php

```diff
@@ -5,4 +5,9 @@
     'master_key'     => null,
     'base_url'       => 'https://nstack.io',
     'version'        => 'v2',
-];
\ No newline at end of file
+    'platform'       => null,
```

Comment:
?

Reply:
should i always assume that it is 'web' ?